src/shapes/SurfaceCircle.js

```diff
@@ -74,6 +74,10 @@ define([
                 set: function(value) {
                     this.stateKeyInvalid = true;
                     this._radius = value;
+                    this.minorRadius = value;
+                    this.majorRadius = value;
+                    this.isPrepared = false;
```

Comment:
All the isPrepared in the shapes don't seem to be used anywhere. Please remove if leftovers.

Reply:
isPrepared is used in SurfaceShape in prepareBoundaries method (see https://github.com/NASAWorldWind/WebWorldWind/blob/6d0fcb824b51817efe032a2b342f31c348d774fc/src/shapes/SurfaceShape.js#L532). For every change of any parameter (radius, heading, etc.) this method is called in order to recalculate boundaries.